lib/webauthn/configuration.rb

```diff
@@ -26,6 +26,7 @@ def self.if_pss_supported(algorithm)
     attr_accessor :encoding
     attr_accessor :origin
     attr_accessor :rp_id
+    attr_accessor :appid
```

Comment:
Adding a config for this makes sense, good call.

However, I think we may want to change the name to a term sounding "less general". For most users coming to implement WebAuthn for the first time, not having read the full spec, a config option named "app id" could sound like something very important to set, while in reality it's not the case for most users.

I will propose something more like `:legacy_u2f_appid` which sounds like something you feel safe ignoring if you don't understand at all :-)

Reply:
Sounds good!